hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsOutputStream.java

```diff
@@ -68,47 +71,69 @@
 public class AbfsOutputStream extends OutputStream implements Syncable,
     StreamCapabilities, IOStatisticsSource {
 
-  private final AbfsClient client;
+  private AbfsClient client;
+
   private final String path;
+
   /** The position in the file being uploaded, where the next block would be
    * uploaded.
    * This is used in constructing the AbfsClient requests to ensure that,
    * even if blocks are uploaded out of order, they are reassembled in
    * correct order.
    * */
   private long position;
+
```

Comment:
Fix unnecessary diffs. Will lead to increased git history load.

Reply:
done